src/Template/Element/Form/meta.twig

```diff
@@ -20,7 +20,7 @@
                     {% for key, val in meta %}
                         <tr>
                             <th>{{ key }}:</th>
-                            <td>{{ Schema.format(val, schema.properties[key]) }}</td>
+                            <td id="{{ key }}">{{ Schema.format(val, schema.properties[key]) }}</td>
```

Comment:
it's not safe to have multiple td with same id. maybe `name` or other attribute could be better, in this context